### PR TITLE
Embed blob sidecars directly under blob_sidecars property in BlockContents

### PR DESCRIPTION
**Pull request by mehdi-aouadi**
## PR Description
Remove redundant `blob_sidecars` property name in `BlockContents` container. This means removing the `BlobSidecars` container and its `BlobSidecarsSchema` (Same for the signed and the signed, blinded ones).
Blob sidecars should be represented as an array under the `blob_sidecars` property in `BlockContents`.
In the current implementation there is a redundant `blob_sidecars` property:
`BlockContents`:
```
{
	"beacon_block": {}
	"blob_sidecars": {
		"blob_sidecars": array of BlobSidecar
	}
}
```

These changes fixes it to become:

```
{
	"beacon_block": {}
	"blob_sidecars": array of BlobSidecar
}
```

These changes also fix `SignedBlockContents`, `BlindedBlockContents` and `SignedBlindedBlockContents`

[Spec reference here](https://github.com/ethereum/beacon-APIs/pull/302)

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [ ] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/BlockContents.java [hunk 1]
```diff
@@ -23,11 +26,11 @@ public class BlockContents {
   private final BeaconBlockDeneb beaconBlock;
 
   @JsonProperty("blob_sidecars")
-  private final BlobSidecars blobSidecars;
+  private final List<BlobSidecar> blobSidecars;
 
   public BlockContents(
       @JsonProperty("beacon_block") final BeaconBlockDeneb beaconBlock,
```
Comment (tbenr) [inline data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/BlockContents.java:32]: `@JsonProperty("block")` ?

File: data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/SignedBlockContents.java [hunk 2]
```diff
@@ -14,20 +14,23 @@
...
 public class SignedBlockContents implements BlockContainer {
   @JsonProperty("signed_beacon_block")
   private SignedBeaconBlockDeneb signedBeaconBlockDeneb;
 
   @JsonProperty("signed_blob_sidecars")
-  private SignedBlobSidecars signedBlobSidecars;
+  private List<SignedBlobSidecar> signedBlobSidecars;
 
   public SignedBlockContents(
       @JsonProperty("signed_beacon_block") final SignedBeaconBlockDeneb signedBeaconBlockDeneb,
```
Comment (tbenr) [inline data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/SignedBlockContents.java:32]: `signed_block`

**Review by tbenr — COMMENTED** [review]
looks generally ok. just a couple of renamings

File: data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/SignedBlockContents.java (hunk 2 above)
Comment (mehdi-aouadi) [reply data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/SignedBlockContents.java:32]: Done

File: data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/BlockContents.java (hunk 1 above)
Comment (mehdi-aouadi) [reply data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/BlockContents.java:32]: Done

**Review by tbenr — APPROVED** [review]
LGTM

**Pull request merged**